packages/tangle/tangle.go

```diff
@@ -381,6 +384,13 @@ func StartSynced(startSynced bool) Option {
 	}
 }
 
+// ForceCacheTime is an Option for the Tangle that allows to override hard coded cache time.
+func ForceCacheTime(forceCacheTime int) Option {
+	return func(options *Options) {
+		options.ForceCacheTime = *database.NewCacheTimeProvider(time.Duration(forceCacheTime))
+	}
+}
+
```

Comment:
Rename option to CacheTimeProvider and accept a `*database.NewCacheTimeProvider` instance instead of an int, when you initialize the `*Options` use a default `CacheTimeProvider` instance with cache disabled (forceCacheTime=-1)